### PR TITLE
Fix OS dependencies for MVAPICH2 with BLCR checkpoint/restart support

### PR DESCRIPTION
**Pull request by besserox**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-1.7-GCC-4.6.3-hwloc-chkpt.eb [hunk 1]
```diff
@@ -23,7 +23,10 @@ dependencies = [('hwloc', '1.5.1')]
 
 # enable checkpointing support
 withchkpt = True
-osdependencies = ['blcr-libs', 'blcr-devel']
+osdependencies = [
+    ('libcr-dev', 'blcr-devel'),
```
Comment (boegel) [inline easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-1.7-GCC-4.6.3-hwloc-chkpt.eb]: why drop `blcr-libs`?

Comment (besserox) [inline easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-1.7-GCC-4.6.3-hwloc-chkpt.eb]: On Debian-like distribution, `libcr-dev` depends on `libcr0`. So it comes by itself if devel is installed.

Isn't it the same on Red Hat-like distribution?

Comment (boegel) [inline easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-1.7-GCC-4.6.3-hwloc-chkpt.eb]: Yes, it is, cfr. http://rpm.pbone.net/index.php3/stat/4/idpl/20828347/dir//com/blcr-devel-0.8.1-1.fc10.x86_64.rpm.html

So, fine.

File: easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-1.7-GCC-4.6.3-hwloc-chkpt.eb [hunk 2]
```diff
@@ -23,7 +23,10 @@ dependencies = [('hwloc', '1.5.1')]
...
+    ('libattr1-dev', 'libattr-devel'),
```
Comment (boegel) [inline easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-1.7-GCC-4.6.3-hwloc-chkpt.eb]: shouldn't we add this as a dep EB-style?

Comment (boegel) [inline easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-1.7-GCC-4.6.3-hwloc-chkpt.eb]: see https://github.com/hpcugent/easybuild-easyconfigs/pull/2706

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/cef9a79dc7b984a5cc95 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
merged via #2707, thanks @besserox!

**Pull request merged**